### PR TITLE
Add Rawhide vs ELN in FAQ

### PR DESCRIPTION
**Pull request by tdawson**
Signed-off-by: Troy Dawson <tdawson@redhat.com>

### CODE REVIEW

**Comment by jkonecny12** [comment]
Looks great to me. Thanks!

**Comment by bookwar** [comment]
There is a dedicated page which describes the buildroot setup of ELN https://docs.fedoraproject.org/en-US/eln/buildroot/

Maybe we can move the table which describes the buildroot configuration from FAQ there. And link to it from FAQ? So that we have one place for all buildroot information.

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -18,4 +18,36 @@ What is the role of ELN with respect to CentOS Stream?::
...
      
-
+What is the differences between Rawhide and ELN?::
+
+    For the majority of packages, there is little or no difference between Rawhide and 
+    ELN packages.  They are the same versions, usually built on the same libraries.  But
+    there are some buildroot differences in global macros that can cause build differences.
+    
+    Note: Many packages have %if fedora / rhel conditions that cause Rawhide and ELN packages
+    to differ.  Listing all of those differences is beyond the scope of this document.
```
Comment (sgallagher) [inline modules/ROOT/pages/faq.adoc:28]:
```suggestion
    Note: Many packages have %if fedora / rhel conditionals in their specfiles that produce differences between the built Rawhide and ELN packages.  Listing all of those differences is beyond the scope of this document.
```

File: modules/ROOT/pages/faq.adoc [hunk 2]
```diff
@@ -18,4 +18,36 @@ What is the role of ELN with respect to CentOS Stream?::
...
+| ppc64le  | -mcpu=power8 -mtune=power8 | -mcpu=power9 -mtune=power9
+| s390x  | -march=zEC12 -mtune=z13 | -march=z14 -mtune=z15
+| x86_64  | - | -march=x86-64-v2
+|===
+
+
+When should I use the rpm macro %\{eln}?::
+
+    Never.  If you think you need to, file a ticket on the 
+    https://github.com/fedora-eln/eln/issues/new/choose[issue tracker].
```
Comment (sgallagher) [inline modules/ROOT/pages/faq.adoc:53]:
```suggestion
    Never.  In nearly all cases, you actually want to use the %\{rhel} macro. If you think you need to use the %\{eln} macro, file a ticket on the https://github.com/fedora-eln/eln/issues/new/choose[issue tracker] with your justification.
```

**Review by sgallagher — CHANGES_REQUESTED** [review]

**Comment by tdawson** [comment]
I used all of Stephens suggestions.
I have moved the buildroot stuff, over to the buildroot page, with links.
I have updated the buildroot page with the latest compiler flag info.

An example is found here:
https://tdawson.fedorapeople.org/eln/public/eln/faq/
https://tdawson.fedorapeople.org/eln/public/eln/buildroot/

File: modules/ROOT/pages/faq.adoc [hunk 3]
```diff
@@ -18,4 +18,22 @@ What is the role of ELN with respect to CentOS Stream?::
 image::ROOT:fedora-eln-centos-stream-rhel.png[]
 
      
-
+What is the differences between Rawhide and ELN?::
```
Comment (sgallagher) [inline modules/ROOT/pages/faq.adoc:21]:
```suggestion
What are the differences between Rawhide and ELN?::
```

**Review by sgallagher — CHANGES_REQUESTED** [review]
Additionally (though this could be a separate change if you prefer), there's a section on the buildroot page that could be updated:

Simply specify `eln` as the target when kicking off a `koji` build using `fedpkg`:
```
fedpkg build [--scratch] --target eln [--srpm [SRPM]] [...]
```

Could be simplified to
Simply specify `eln` as the release when kicking off a `koji` build using `fedpkg`:
```
fedpkg --release eln build [--scratch] [--srpm [SRPM]] [...]
```


I mostly note this because the existing example is actually wrong. You would need to specify both `--target eln` and `--skip-nvr-check` for it to work properly, whereas `--release eln` handles it correctly.

**Review by sgallagher — APPROVED** [review]
LGTM

**Pull request merged**